src/client/eppo-client.ts

```diff
@@ -866,13 +866,18 @@ export default class EppoClient {
    *
    * @param subjectKey an identifier of the experiment subject, for example a user ID.
    * @param subjectAttributes optional attributes associated with the subject, for example name and email.
-   * @param banditActions
+   * @param banditActions optional attributes associated with the bandit actions
+   * @param salt a salt to use for obfuscation
    */
   getPrecomputedConfiguration(
     subjectKey: string,
     subjectAttributes: Attributes | ContextAttributes = {},
     banditActions: Record<FlagKey, BanditActions> = {},
+    salt?: string,
   ): string {
+    if (!salt) {
+      throw new Error('[Eppo SDK] Salt is required for precomputed configuration');
```

Comment:
This error would only show up if using the common sdk directly. We will ensure that a salt is provided in the node server sdk

Reply:
After further consideration, let's not throw errors if we can help it